core/src/main/java/hudson/model/JDK.java

```diff
@@ -54,10 +54,22 @@
 public final class JDK extends ToolInstallation implements NodeSpecific<JDK>, EnvironmentSpecific<JDK> {
 
     /**
-     * Name of the “default JDK”, meaning no specific JDK selected.
+     * Name of the “System JDK”, which is just the JDK on Jenkins' $PATH.
      * @since 1.577
      */
-    public static final String DEFAULT_NAME = "(Default)";
+    public static final String DEFAULT_NAME = "(System)";
+
+    public static boolean isDefaultName(String name) {
```

Comment:
Consider `@Restricted(NoExternalUse.class)`, I see no reason to make this public API.